src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -113,9 +105,12 @@ export interface IChartContainerProps {
     onResetSelection?: any;
     onDeleteChart: (chartMeta: ChartMeta) => void;
     onChangeChartType: (chartMeta: ChartMeta, newChartType: ChartType) => void;
-    onToggleLogScale?: any;
+    onToggleLogScale: (chartMeta: ChartMeta) => void;
```

Comment:
just changed this to be non-optional

Reply:
@kalletlak what are your thoughts? it seems like this change doesn't affect compiling